Could not resolve dependencies for project fr.acinq.eclair:eclair-front_2.13:jar:0.5.1-SNAPSHOT

Describe the issue:
Building eclair fails

What behaviour did you expect? 
Successful build

What was the actual behaviour (provide screenshots if the issue is GUI-related)?

$ mvn package -Dmaven.test.skip=true
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.google.inject.internal.cglib.core.$ReflectUtils$1 (file:/usr/share/maven/lib/guice.jar) to method java.lang.ClassLoader.defineClass(java.lang.String,byte[],int,int,java.security.ProtectionDomain)
WARNING: Please consider reporting this to the maintainers of com.google.inject.internal.cglib.core.$ReflectUtils$1
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Build Order:
[INFO] 
[INFO] eclair_2.13                                                        [pom]
[INFO] eclair-core_2.13                                                   [jar]
[INFO] eclair-front                                                       [jar]
[INFO] eclair-node                                                        [jar]
[INFO] eclair-node-gui                                                    [jar]
[INFO] 
[INFO] --------------------< fr.acinq.eclair:eclair_2.13 >---------------------
[INFO] Building eclair_2.13 0.5.1-SNAPSHOT                                [1/5]
[INFO] --------------------------------[ pom ]---------------------------------
[INFO] 
[INFO] --- git-commit-id-plugin:2.2.3:revision (default) @ eclair_2.13 ---

<snip the parts that worked>

[INFO] -----------------< fr.acinq.eclair:eclair-front_2.13 >------------------
[INFO] Building eclair-front 0.5.1-SNAPSHOT                               [3/5]
[INFO] --------------------------------[ jar ]---------------------------------
Downloading from sonatype snapshots: https://oss.sonatype.org/content/repositories/snapshots/fr/acinq/eclair/eclair-core_2.13/0.5.1-SNAPSHOT/maven-metadata.xml
Downloading from sonatype snapshots: https://oss.sonatype.org/content/repositories/snapshots/fr/acinq/eclair/eclair-core_2.13/0.5.1-SNAPSHOT/eclair-core_2.13-0.5.1-SNAPSHOT-tests.jar
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for eclair_2.13 0.5.1-SNAPSHOT:
[INFO] 
[INFO] eclair_2.13 ........................................ SUCCESS [  6.605 s]
[INFO] eclair-core_2.13 ................................... SUCCESS [ 53.847 s]
[INFO] eclair-front ....................................... FAILURE [  2.246 s]
[INFO] eclair-node ........................................ SKIPPED
[INFO] eclair-node-gui .................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:02 min
[INFO] Finished at: 2020-12-29T09:21:38-05:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project eclair-front_2.13: Could not resolve dependencies for project fr.acinq.eclair:eclair-front_2.13:jar:0.5.1-SNAPSHOT: Could not find artifact fr.acinq.eclair:eclair-core_2.13:jar:tests:0.5.1-SNAPSHOT in sonatype snapshots (https://oss.sonatype.org/content/repositories/snapshots/) -> [Help 1]


How reliably can you reproduce the issue, what are the steps to do so?
Every time

What version of Eclair are you using, where did you get it (website, self-compiled, etc)?
0.5.1-Snapshot
self-compiled

What environment are you observing the error on (OS, JDK version)?
Debian-Linux 10, kernel 4.19.0, OpenJDK 11.0.9

--- a/BUILD.md
+++ b/BUILD.md
@@ -19,6 +19,7 @@ mvn package
 ```
 
 Notes:
+
 - This command will build all modules (core, node, gui).
 - If the build fails, you may need to clean previously built artifacts with the `mvn clean` command.
 - Some of the tests use [Docker](https://www.docker.com/), so make sure your docker daemon is running.
@@ -30,12 +31,6 @@ Running tests takes time. If you want to skip them, use `-DskipTests`:
 
 ```shell
 mvn package -DskipTests
-```
-
-You can even skip the tests compilation with `maven.test.skip`:
-
-```shell
-mvn package -Dmaven.test.skip=true
 ```
 
 ### Run tests

